docs/book/src/reference/versions.md

```diff
@@ -67,10 +67,11 @@ The table below documents support matrix for Cluster API versions (versions olde
 
 | Minor Release | Status                  | Supported Until (including maintenance mode)                                                |
 |---------------|-------------------------|---------------------------------------------------------------------------------------------|
-| v1.11.x       | Under development       |                                                                                             |
+| v1.12.x       | Under development       |                                                                                             |
```

Comment:
Is this correct? We just released v1.10.x and are now working on v1.11

Reply:
I'm not sure if we have to update this table at all

Reply:
@sbueringer my idea was to get this page ready for when we will release, so we do not forget (see https://github.com/kubernetes-sigs/cluster-api/pull/12124)

But happy to rollback if you prefer

Reply:
I see your point. It's just that when folks look at this doc on main it's very confusing. Do we know how we did this in the past? (I would expect we have some tracking in our release tasks)

Reply:
I would say: 

Like when folks look at code in main they get the in progress implementation for the next release,
they also get in progress docs for the next release.

To your point, as far as I remember, it is our a common practice to work on the book on main documenting what will become true when we release.

We do this for every new feature, for every breaking change etc, and this already started happening also in this release: we have a page where we are documenting changes for providers, we already changed contract documentation to align to v1beta2 etc.
(e.g also in this page we are already stating v1beta2 as current APi, v1beta1 as deprecated)

Reply:
In the release book the task is tracked as https://github.com/kubernetes-sigs/cluster-api/tree/main/docs/release/role-handbooks/communications#update-supported-versions (I guess)

Reply:
@sbueringer, as discussed I have dropped the line for 1.12 release under development